plugin/server/metrics_service.py

```
@@ -0,0 +1,247 @@
+                    return [self._metrics_to_dict(history[-1])]
+                # 调试：记录为什么找不到指标
+                available_ids = list(self._metrics_history.keys())
+                logger.debug(
+                    f"Metrics not found for plugin_id '{plugin_id}'. "
+                    f"Available plugin_ids in metrics_history: {available_ids}"
+                )
+                return []
+            else:
+                # 返回所有插件的最新指标
+                result = []
+                for pid, history in self._metrics_history.items():
```

Ruff (0.14.10)

Loop control variable `pid` not used within loop body

Rename unused `pid` to `_pid`

(B007)